clusters.cnp.example.org/.crossplane/allocatedcluster.ui-schema.yaml

```diff
@@ -0,0 +1,83 @@
+initialConfig: true
+configSections:
+- title: CNP Kubernetes Cluster
+  description: Provide the following details to provision a Kubernetes cluster
+  items:
+  - name: region
```

Comment:
`region` is currently a literal in the eks.clusters.cnp.example.org composition.yaml, so we can remove this for now.

Reply:
I'd prefer to merge and follow up on these UI metadata changes in another PR.